os/booting-on-cloudca.md

```diff
@@ -0,0 +1,260 @@
+# Running CoreOS on cloud.ca
+
+[cloud.ca](http://www.cloud.ca) is a regional self-service IaaS platform
+designed for workspaces, applications and data. It delivers scalable,
+high-performance and resilient cloud infrastructure at competitive utility
+costs. CloudOps provides [cloud.ca as a service](http://www.cloudops.com/services/managed-services/managed-cloud/)
+for businesses requiring that all or some of their data remain in Canada, for
+reasons of compliance, performance, cost or privacy. cloud.ca's platform can be
+used as a standalone IaaS and also fits into CloudOps’ hybrid cloud services,
+allowing a mix of private cloud and other public cloud infrastructures such as
+AWS.
+
+
+## Choosing a Channel
+
+CoreOS is designed to be [updated automatically][update-docs] with different
```

Comment:
CoreOS is designed to be automatically updated, with three channels of different update schedules: Stable, Beta, and Alpha.